detect_variants/add_bam_readcount_to_vcf.cwl

```diff
@@ -18,7 +18,6 @@ inputs:
             items: File
         inputBinding:
             prefix: ""
-            itemSeparator: ","
```

Comment:
For tumor-normal multi-sample workflows we probably still need this, right? Does it cause an error in the tumor-only workflow to run it with the itemSeparator?

Reply:
I was kind of wondering about that... I can take this commit out.  If it really is causing issues we should find find a different solution that works for both.